core/store/migrate/migrations/0161_add_cosmos.sql

```diff
@@ -0,0 +1,52 @@
+-- +goose Up
+
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE cosmos_chains (
+    id text PRIMARY KEY,
+    cfg jsonb NOT NULL DEFAULT '{}'::jsonb,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL,
+    enabled boolean NOT NULL DEFAULT true
+);
+
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE cosmos_msgs (
+    id BIGSERIAL PRIMARY KEY,
+    cosmos_chain_id text NOT NULL REFERENCES cosmos_chains(id) ON DELETE CASCADE,
+    contract_id text NOT NULL,
+    raw bytea NOT NULL,
+    state text NOT NULL,
+    tx_hash text,
+    created_at timestamp with time zone NOT NULL,
+    updated_at timestamp with time zone NOT NULL,
+    type text NOT NULL DEFAULT '/cosmos.wasm.v1beta1.MsgExecuteContract'::text,
```

Comment:
```suggestion
    type text NOT NULL DEFAULT '/cosmwasm.v1.MsgExecuteContract'::text,
```
This is now the official type https://github.com/cosmos/cosmjs/blob/33271bc51c/packages/cosmwasm-stargate/src/modules/wasm/messages.ts#L70